bc_obps/reporting/service/sync_validation_service.py

```diff
@@ -0,0 +1,47 @@
+from reporting.models import ReportVersion
+from service.reporting_year_service import ReportingYearService
+
+
+class SyncValidationService:
+    """
+    Service to validate whether sync operations are allowed for a given report version.
+    """
+
+    @classmethod
+    def is_sync_allowed(cls, version_id: int) -> bool:
+        """
+        Check if syncing with admin data is allowed for the given report version.
+
+        Sync is NOT allowed if:
+        1. The report is not for the current reporting year
+        2. The operation is not currently owned by the operator who filed the report
```

Comment:
I think we need to add a 3rd check here to make sure that a facility (in the case of an LFO) hasn't been transferred (while the operation itself remains with the original operator). This would be relevant for the "Review Facility Info" page (from the ticket)